### DISCUSSION TITLE
如何让文件只在子分类内显示？系统设置无效

### DISCUSSION THREAD

**Discussion by ghost** [zotero-chinese/wiki#309] [求助 / Q&A] [answered]
已经尝试过不勾选“显示子分类中的条目”，但条目依旧在所有分类内显示。

![zotero_OmeQZBpqOM](https://github.com/user-attachments/assets/b807325c-3587-4d42-878b-4efc41417351)

**Comment by northword** [comment]
你这些条目同时存在于两个分类，在这里查看：

![PixPin_2024-11-26_16-47-45](https://github.com/user-attachments/assets/71b4be5b-1a4c-4602-a5dc-8821acb9b210)

**Reply by ghost** [reply]
我大概能理解了。有几个新问题：
- 我第二个“test”文件夹是建在我第一个“学习”文件夹下的，请问zotero的逻辑是将它们视作不同的文件夹而非父子文件夹吗？
- 尝试操作了一下。每篇文章都需要手动调整，有没有更快捷的方式？（类似windows下的文件夹管理）

**Reply by northword** [reply, marked as answer]
它的英文是 Collection，合集，一个条目可以包含在多个合集。

所以和文件夹那种明确的单一从属关系不太一样。

(我一直认为这个分类翻译的不合理，但如果要改工作量太大了，而且其他同学认为分类更好理解)

多选，按shift拖动是移动。

**Reply by ghost** [reply]
我搞懂你的意思了。所以我想到的处理方案有：
- 预先建好子文件夹，文件初始拖入子文件夹内，这样就可以开关显示了。
- 父分类内已有的文件拖入子文件夹后，再在父分类内删除条目。
感觉这个逻辑有些画蛇添足了，不如直接拖入之后默认在父文件夹内不显示。
感谢回答。
